torch/csrc/jit/passes/quantization.cpp

```diff
@@ -87,8 +164,89 @@ void InsertObserverNodes(std::shared_ptr<Graph>& graph, Node* observer_node) {
   }
 }
 
-void InsertFakeQuantNodes(std::shared_ptr<Graph>& graph) {
-  throw std::runtime_error("Pass not implemented yet!");
+void InsertQuantDequantNodes(std::shared_ptr<Graph>& graph) {
+  std::stack<Block*> blocks_to_visit;
+  blocks_to_visit.push(graph->block());
+  // For storing quantizable values - node pairs that are external
+  // or intermediate inputs to quantizable nodes
+  std::vector<std::pair<Value*, Node*>> quantInputs;
```

Comment:
We could use `Use*` instead of `pair<Value*, Node*>`, I believe it would express the logic better.